Makefile

```diff
@@ -1,8 +1,10 @@
 WEBMIN_FW_TCP_INCOMING = 22 80 443 3306 12320 12321 12322
 
-COMMON_OVERLAYS = adminer tkl-webcp lighttpd
+COMMON_OVERLAYS = adminer tkl-webcp 
 COMMON_CONF = adminer-lighttpd adminer-mysql tkl-webcp
 
+include $(FAB_PATH)/common/mk/turnkey/lighttpd.mk
+include $(FAB_PATH)/common/mk/turnkey/php-fpm.mk
 include $(FAB_PATH)/common/mk/turnkey/php.mk
```

Comment:
FWIW `php-fpm.mk` includes `php.mk` (see https://github.com/turnkeylinux/common/blob/master/mk/turnkey/php-fpm.mk#L3) so this line can be removed.

Reply:
I went through and changed the code to remove the extra include php.mk file from common.